tests/suites/deploy/deploy_default_series.sh

```diff
@@ -7,14 +7,27 @@ run_deploy_default_series() {
 	ensure "${model_name}" "${file}"
 
 	juju model-config default-series=bionic
-	juju deploy ubuntu
-	juju deploy cs:ubuntu csubuntu
+	juju deploy ubuntu --storage "files=tmpfs"
+	wait_for "ubuntu" "$(idle_condition "ubuntu")"
+	series=$(juju status --format=json | jq ".applications.ubuntu.series")
+	echo "$series" | check "bionic"
 
-	ubuntu_series=$(juju status --format=json | jq ".applications.ubuntu.series")
-	echo "$ubuntu_series" | check "bionic"
+	destroy_model "${model_name}"
+}
 
-	csubuntu_series=$(juju status --format=json | jq ".applications.csubuntu.series")
-	echo "$csubuntu_series" | check "bionic"
+run_deploy_default_series_cs() {
+	echo
+
+	model_name="test-deploy-default-series-cs"
+	file="${TEST_DIR}/${model_name}.log"
+
+	ensure "${model_name}" "${file}"
+
+	juju model-config default-series=bionic
+	juju deploy cs:ubuntu --storage "files=tmpfs"
```

Comment:
does this actually use the latest ubuntu charm when coming from cs?

Reply:
Yep!

Reply:
@hpidcock cs is backed by charmhub via the compatibility api. If a charm has started life as cs, any revisions released to latest/stable in charmhub, is available via cs